graylog2-web-interface/src/components/sidecars/configurations/CopyConfigurationModal.jsx

```diff
@@ -46,16 +46,20 @@ class CopyConfigurationModal extends React.Component {
     const configuration = this.state;
 
     if (!configuration.error) {
-      this.props.copyConfiguration(this.props.id, this.state.name, this._saved);
+      this.props.copyConfiguration(this.state.id, this.state.name, this._saved);
     }
   };
 
   _changeName = (event) => {
     const nextName = event.target.value;
     this.setState({ name: nextName });
-    this.props.validateConfiguration(nextName).then(validation => (
-      this.setState({ error: validation.error, error_message: validation.error_message })
-    ));
+    this.props.validateConfiguration({ name: nextName }).then((validation) => {
```

Comment:
Trying to clone a valid configuration fails with:

```JSON
{"errors":{"template":["Collector template cannot be empty."],"color":["Collector color cannot be empty."],"collector_id":["Associated collector ID cannot be empty."]},"failed":true}
```

I guess the error is here, as we only pass the name into the validation function (which defaults every property into an empty string).